graylog2-web-interface/src/views/components/searchbar/saved-search/SavedSearchControls.jsx

```diff
@@ -228,50 +205,48 @@ class SavedSearchControls extends React.Component<Props, State> {
     );
 
     return (
-      <NewViewLoaderContext.Consumer>
-        {(loadNewView) => (
-          <div className="pull-right">
-            <ButtonGroup>
-              <>
-                <Button title={title} ref={(elem) => { this.formTarget = elem; }} onClick={this.toggleFormModal}>
-                  <Icon style={{ color: savedSearchColor }} name={savedSearchStyle} /> Save
-                </Button>
-                {savedSearchForm}
-              </>
-              <Button title="Load a previously saved search"
-                      onClick={this.toggleListModal}>
-                <Icon name="folder-o" /> Load
-              </Button>
-              {savedSearchList}
-              <DropdownButton title={<Icon name="ellipsis-h" />} id="search-actions-dropdown" pullRight noCaret>
-                <MenuItem onSelect={this.loadAsDashboard}><Icon name="dashboard" /> Export to dashboard</MenuItem>
-                <MenuItem onSelect={this.toggleCSVExport}><Icon name="cloud-download" /> Export to CSV</MenuItem>
-                <MenuItem disabled={disableReset} onSelect={() => loadNewView()} data-testid="reset-search">
-                  <Icon name="eraser" /> Reset search
-                </MenuItem>
-                <MenuItem divider />
-                <MenuItem onSelect={this.toggleShareSearch} title="Share search" disabled={!(view && view.id) || !_isAllowedToEdit(view, currentUser)}>
-                  <Icon name="share-alt" /> Share
-                </MenuItem>
-              </DropdownButton>
-              {csvExport}
-              {shareSearch}
-            </ButtonGroup>
-          </div>
+      <CurrentUserContext.Consumer>
+        {(currentUser) => (
+          <NewViewLoaderContext.Consumer>
+            {(loadNewView) => (
+              <div className="pull-right">
+                <ButtonGroup>
+                  <>
+                    <Button title={title} ref={(elem) => { this.formTarget = elem; }} onClick={this.toggleFormModal}>
```

Comment:
Since we're revising this component already, could we use [`createRef`](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs)?